src/main/resources/application.yml

```diff
@@ -134,3 +134,8 @@ reports:
   non-associations:
     reconciliation:
       page-size: 40
+
+hmpps.sqs:
+  queues:
+    location:
+      errorVisibilityTimeout: 30
```

Comment:
Also adjust queue retries so it doesnt try again too quickly. These can fail due to a missing parent location